chart-app/src/components/ChartApp.vue

```diff
@@ -3,28 +3,29 @@
         <h1>{{msg}}</h1>
 
         <!--    This div is the container for the chart component-->
-        <div class="chart-container">
+        <div id="chart-container" class="chart-container" hidden>
```

Comment:
```suggestion
        <div id="chart-container" class="chart-container" :class="{'hidden': hideChart}">
```
(See my comments below)